sbb/agora2mets.pl

```diff
@@ -0,0 +1,488 @@
+#!/opt/ActivePerl-5.16/bin/perl
```

Comment:
Goobi contrib tools must not be run on every environment without modification. It's fine for me.